src/fidesops/service/connectors/saas_connector.py

```diff
@@ -87,6 +88,7 @@ def send(
         """
         try:
             prepared_request = self.get_authenticated_request(request_params)
+            log_request_for_debugging(prepared_request)  # Dev mode only
             response = self.session.send(prepared_request)
```

Comment:
what does everyone think about logging response.json as well? I'm usually doing this manually for debugging purposes

Reply:
that's a good idea -

Reply:
logging response._content in case the response isn't json -

Reply:
logging response._content in case the response isn't json -